airbyte-integrations/bases/debezium/src/main/java/io/airbyte/integrations/debezium/internals/DebeziumRecordIterator.java

```diff
@@ -203,4 +200,35 @@ private void throwExceptionIfSnapshotNotFinished() {
     }
   }
 
+  @VisibleForTesting
+  protected Long getHeartbeatPosition(final ChangeEvent<String, String> heartbeatEvent) {
+    try {
+      if (heartbeatEvent == null) {
+        return null;
+      }
+
+      final Class<? extends ChangeEvent> eventClass = heartbeatEvent.getClass();
+      final Field f;
+      if (heartbeatEventSourceField.containsKey(eventClass)) {
+        f = heartbeatEventSourceField.get(eventClass);
+      } else {
+        f = eventClass.getDeclaredField("sourceRecord");
+        f.setAccessible(true);
+        heartbeatEventSourceField.put(eventClass, f);
```

Comment:
I still don't fully understand what `heartbeatEventSourceField` is, how is intended to be used, ...
I see it's a property for the class `DebeziumRecordIterator` but we don't have any description of it.
That would help to review/modify the code in the future.

Reply:
Could you add a description to the variable `heartbeatEventSourceField`?